compiler/rustc_ast_lowering/src/path.rs

```diff
@@ -16,8 +16,8 @@ use rustc_hir::GenericArg;
 use rustc_middle::span_bug;
 use rustc_span::symbol::{kw, sym, Ident};
 use rustc_span::{BytePos, DesugaringKind, Span, Symbol, DUMMY_SP};
-
 use smallvec::{smallvec, SmallVec};
+use tracing::{debug, instrument};
```

Comment:
I think this separation is because the `rustc_*` are in the workspace while smallvec and tracing are not. I think stylistically the separation is slightly more readable

Reply:
Ok.